index.ts

```diff
@@ -118,12 +118,12 @@ export function encodeRunestoneUnsafe(runestone: RunestoneSpec): {
   }));
 
   let etching: Option<Etching> = None;
-  let etchingCommitment: string | undefined = undefined;
+  let etchingCommitment: Buffer | undefined = undefined;
```

Comment:
is this change related?

Reply:
oh nvm, i see that it shadows the input arg name

not related to this change, but we should choose a different name here to not conflict with the arg name for clarity

Reply:
Not arg name above, but the return type